BLSQ/iaso
# Wc2-94: csv/xls exports of list of storages

The list of storage devices can now be exported in CSV and XLSX formats
## Self proof reading checklist


## How to test

1) Make sure you have Storage devices (use Django admin to populate)
2) Generate downloads with URLs such as `/api/storage/?xlsx=true`, `/api/storage/?csv=true`, `/api/storage/?xlsx=true&status=BLACKLISTED` and `/api/storage/?csv=true&order=type`

(once the download buttons are implemented in the frontend, we can test more easily and realistically (end-to-end feature test) 

## Print screen / video
[recording-2022-11-18-17-41-04.webm](https://user-images.githubusercontent.com/25134301/202756525-76296047-62b2-4b62-8994-913b3b40d5f3.webm)
